ChangeLog

```diff
@@ -29,6 +29,10 @@
 - Improved __repr__ methods in pygambit for game-related classes
 - Further extension of test suite for mixed behavior profiles to cover new indexing and profile
   order consistency for payoff-related calculations
+- Overhaul of caching in MixedBehaviorProfile: std::map is used as the container for cached
```

Comment:
Don't think we need this much detail - could just write "Changed implementation of caching in implementation of MixedBehaviorProfile (#387).  Especially for items which aren't user-facing brevity's helpful, and especially when we can just link it to the relevant issue for more information.

Reply:
I generally agree, but actually in this case we didn't even mention maps in the issue. Still I'll be much briefer.